modules/OpenWeather/module.json

```
@@ -6,7 +6,7 @@
 	"homepage": "http://razberry.z-wave.me",
 	"icon": "icon.png",
 	"moduleName":"OpenWeather",
-	"version": "1.2.0",
+	"version": "1.2.1",
```

```suggestion
	"version": "1.3.0",
```

This is not a minor change